macos/video-toolbox/src/video_encoder.rs

```diff
@@ -27,6 +27,7 @@ pub struct VideoEncoderConfig {
     pub fps: f64,
     pub codec: VideoEncoderCodec,
     pub input_format: VideoEncoderInputFormat,
+    pub high_availability_mode: bool,
```

Comment:
In this repo, "high availability" is meaningless. I'd suggest just making this an optional max key frame interval:

```suggestion
    pub max_keyframe_interval: Option<i32>,
```

Reply:
👍